docs/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 
 ## 6.10.0 <span className="changelog-date">(2020-05-10)</span>
 
+* Assign and display mods to an armour slot they will fit with in LO
+
```

Comment:
This should be removed

Reply:
I figured this would be the case but I wasn't 100% sure on changing history, its done now.